onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -122,6 +122,13 @@ void SplitTestAxis0EqualSplit(bool use_opset_13 = false) {
              {kTensorrtExecutionProvider},  // is_tensorrt_supported
              false,                         // expect_failure
              use_opset_13);                 // split_as_input
+#ifdef USE_COREML
```

Comment:
actually, should we run this for other EPs as well?

see Scott's comment:
https://github.com/microsoft/onnxruntime/pull/18358/files/0d96ebe4e165d138bedb939f0a4ec344f37ad693#r1400070141

maybe we can get rid of the `use_opset_13` parameter and just have a special case for BFloat16 here to skip `split_as_input = false`.

```
// BFloat16 added in opset 13
if constexpr (!std::is_same_v<T, BFloat16>) {
  RunTest(..., false /* split_as_input */);
}

RunTest(..., true /* split_as_input */);
```

Reply:
cool, updated!